### PR TITLE
Adds a comments widget to core

### PR DESCRIPTION
**Pull request by davejlong**
Widget to display comment style information (avatar, user name, and text)

**Usage:**

Broadcast data in the following format from jobs:

```elixir
%{
  avatar: "http://placehold.it/50x50",
  name: "John Smith",
  quote: "Dang! Kitto is super awesome!"
}
```

To display in a dashboard:

```html
<div data-widget="Comments"
     data-source="blog_comments"
     data-title="Blog"
     data-moreinfo="mycoolblog.com"></div>
```

### CODE REVIEW

File: installer/templates/new/widgets/comments/comments.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    h3 {
+      img {
+        border: 1px solid $title-color;
+        margin-right: 0.5rem;
+      }
+    }
+  }
+
+  .title { color: $title-color; }
+  .more-info, .updated-at { color: $moreinfo-color; }
```
Comment (zorbash) [inline installer/templates/new/widgets/comments/comments.scss]: Each selector in a comma sequence should be on its own single line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#singlelineperselector)

File: installer/templates/new/widgets/comments/comments.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+// Widget-Comments styles
+// ----------------------------------------------------------------------------
+.widget-comments {
+  background-color: $background-color;
+
+  .comment-container {
+    h3 {
+      img {
+        border: 1px solid $title-color;
+        margin-right: 0.5rem;
```
Comment (zorbash) [inline installer/templates/new/widgets/comments/comments.scss]: `0.5` should be written without a leading zero as `.5`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero)

File: installer/templates/new/widgets/comments/comments.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+// ----------------------------------------------------------------------------
+// Widget-Comments styles
+// ----------------------------------------------------------------------------
+.widget-comments {
+  background-color: $background-color;
+
+  .comment-container {
+    h3 {
+      img {
+        border: 1px solid $title-color;
```
Comment (zorbash) [inline installer/templates/new/widgets/comments/comments.scss:19]: Properties should be ordered margin-right, border

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: installer/templates/new/widgets/comments/comments.scss [hunk 4]
```diff
@@ -0,0 +1,26 @@
+// ----------------------------------------------------------------------------
+// Sass declarations
+// ----------------------------------------------------------------------------
+$background-color:  #eb9c3c;
+
+$title-color:       rgba(255, 255, 255, 0.7);
+$moreinfo-color:    rgba(255, 255, 255, 0.7);
```
Comment (zorbash) [inline installer/templates/new/widgets/comments/comments.scss]: `0.7` should be written without a leading zero as `.7`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero)

File: installer/templates/new/widgets/comments/comments.scss [hunk 5]
```diff
@@ -0,0 +1,26 @@
+// ----------------------------------------------------------------------------
+// Sass declarations
+// ----------------------------------------------------------------------------
+$background-color:  #eb9c3c;
+
+$title-color:       rgba(255, 255, 255, 0.7);
```
Comment (zorbash) [inline installer/templates/new/widgets/comments/comments.scss]: `0.7` should be written without a leading zero as `.7`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero)

**Comment by Zorbash** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 5 possible new issues (including those that may have been commented here).

You can see more details about this review at https://ebertapp.io/github/kittoframework/kitto/pulls/61.

**Comment by zorbash** [comment]
This is a neat pull request with documentation and everything, but i feel like a comments widget does not belong to the core as it doesn't seem like a building block widget. We should definitely add a reference in the [additional-widgets](https://github.com/kittoframework/kitto/wiki/Widget-and-Job-Directory) section in the wiki.

**Comment by davejlong** [comment]
I will add it to the widget repository.

**Pull request closed without merging**